src/components/ContributionAssistantDrawCanvas.vue

```diff
@@ -18,6 +18,10 @@
       image: {
         type: Image,
         default: null
+      },
+      seedCrops: {
```

Comment:
could we find another name ? when I read this I imagine myself in an farmer's crop/field ^^
the backend sends `bounding_box` arrays. 
- maybe replace `rectangles` by `boundingBoxes`
- and `seed` by `initFromServer` or something like this ?

Reply:
honestly it's not important, we can refactor later if you feel like/find a better name ^^